.github/workflows/R-CMD-check.yaml

```diff
@@ -107,9 +107,9 @@ jobs:
 
       - name: Install TensorFlow
         run: |
-          reticulate::conda_create('r-reticulate', packages = c('python==3.7', 'numpy==1.19'))
+          reticulate::conda_create('r-reticulate', packages = c('python==3.7'))
           remotes::install_local()
-          keras::install_keras(tensorflow = Sys.getenv('TF_VERSION'), extra_packages = c('IPython', 'requests', 'certifi', 'urllib3', 'tensorflow-probability==0.7.0'))
+          keras::install_keras(tensorflow = Sys.getenv('TF_VERSION'), extra_packages = c('IPython', 'requests', 'certifi', 'urllib3', 'tensorflow-probability==0.7.0', 'numpy==1.16.4'))
```

Comment:
any particular reason for installing numpy in the `install_keras()`  step and not in the `conda_create()` step?

Reply:
Yup! When I put it in conda create it installed numpy 1.20 later, I assume because the other packages suggest a later install of numpy